recipe/meta.yaml

```diff
@@ -8,9 +8,11 @@ source:
   fn: sympy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/sympy/sympy-{{ version }}.tar.gz
   md5: c410a9c2346878716d16ec873d72e72a
+  patches:
+  - patches/13276.patch
```

Comment:
I see the appeal for a folder if you have tons of patches,
but in conda-forge we usually do not use a folder for the patches b/c we have 1-3 max.

Anyway, I'll let @conda-forge/sympy team decide if they want the folder or not.

Reply:
I was not aware of this. Fixed now.

Reply:
It depends on the preference of the maintainers. I like a `patches` folder.

Reply:
To be honest I don't have a strong preference. Just pointed out what is more common.
Sorry @saraedum for the extra work. Let's wait for @conda-forge/sympy team decided what they want here :wink:

Reply:
It's really not that important, but I slightly prefer to not have a folder because it doesn't encourage patches, which as I noted in my other comment, there should be minimal of.